src/MainWindow.vala

```diff
@@ -126,6 +134,19 @@ public class AppCenter.MainWindow : Hdy.ApplicationWindow {
 
         title = _(Build.APP_NAME);
 
+        toast = new Granite.Widgets.Toast ("");
+        toast.set_default_action (_("Open"));
+
+        toast.default_action.connect (() => {
+            if (last_installed_package != null) {
+                try {
+                    last_installed_package.launch ();
+                } catch (Error e) {
+                    warning ("Failed to launch %s: %s".printf (last_installed_package.get_name (), e.message));
```

Comment:
Hm, I wonder if this should throw a dialog. If a user clicked the open button and then nothing happened, that seems bad. Thoughts @cassidyjames?

Reply:
For reference: This code is mostly a copy from `AbstractAppContainer.launch_package_app()`

Reply:
@danrabbit I'm indifferent. It probably makes sense to do what we do in other instances in AppCenter, and then we can file an issue to throw a dialog on failure.